optional_plugins/runner_remote/setup.py

```diff
@@ -13,12 +13,13 @@
 # Copyright: Red Hat Inc. 2017
 # Author: Cleber Rosa <crosa@redhat.com>
 
-import sys
+import distro
 
 from setuptools import setup, find_packages
 
 
-if sys.version_info[0] == 3:
+if distro.linux_distribution()[0] == 'Fedora' and \
```

Comment:
This could be something like:

```python
from avocado.utils import distro
detected = distro.detect()
if detected.name == 'fedora' and int(detected.version) >= 29:
   ...
```